Decorator/MessageBox.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Decorator
+{
+    class MessageBox
+    {
+        private string text;
+        public string Text { get => text; set => text = new EmojiDecorator().Filter(value); }
```

Comment:
`get` and `set` is better to put on the different strings for readability

Reply:
done